src/Data/Aeson/TypeScript/Types.hs

```diff
@@ -93,12 +93,14 @@ data FormattingOptions = FormattingOptions
   -- ^ Function applied to generated interface names
   , typeNameModifier :: String -> String
   -- ^ Function applied to generated type names
+  , exportTypes :: Bool
```

Comment:
Could you add a documentation string here?

Reply:
Also curious about the need for this -- I always put my types in `.d.ts` files, is there some reason you want to put them in an ES6 module?

Reply:
I generate automatically not only the types, but also automatically generated code that builds a data structure that holds `fetch` invocations with the right request and response type, through https://github.com/bitonic/solga/tree/francesco/solga-typescript . I store the generated types alongside this definition, and therefore I need the exports.